search-app/src/cmr/search/services/acl_service.clj

```diff
@@ -119,8 +119,23 @@
   [context concepts]
   (when (seq concepts)
     (if (tc/echo-system-token? context)
-      ;;return all concepts if running with the system token
-      concepts
+     ;;TODO JYNA temporarily forcing it past system token
+     ; concepts
```

Comment:
This isn't correct.  If you have a system token then all collections should be shown and not filtered out so I think you want to just pass back the concepts.  The you can get rid of your TODO comment.

Then delete lines 124 to 138 as they seem to be identical to 139-153.

Reply:
This was temp logic to force a certain pathway I wanted in local. Has been reverted in current PR